action_server/src/action_server/actions/example_action.py

```diff
@@ -0,0 +1,133 @@
+from action import Action, ConfigurationData
+from robot_skills.util.entity import Entity
+
+
+class ExampleAction(Action):
+    """
+    The ExampleAction class explains how Actions, as handled by the TaskManager, work.
+    """
+    def __init__(self):
+        # Call the base class constructor
+        Action.__init__(self)
+        """
+        The base class constructor initializes the following fields:
+         - self._config_result = ConfigurationResult()
+         - self._execute_result = ActionResult()
+        Their fields should be assigned meaningful values for the caller to give decent feedback to the user.
```

Comment:
Their fields should be assigned meaningful values **in the _configure() and _start() methods respectively, so** the caller **can** give decent feedback to the user.

I was confused if these should be given values here in the constructor?